nff/io/ase_calcs.py

```diff
@@ -283,10 +299,8 @@ def offset_energy(self, atoms, energy: Union[float, np.ndarray]):
         for ele, num in ads_count.items():
             ref_en += num * stoidict.get(ele, 0.0)
         ref_en += stoidict.get("offset", 0.0)
-        if self.offset_units == "atomic":
```

Comment:
@sauradeep93 maybe some unintentional change?

Reply:
Same here, maybe these changes weren't reflected in @sauradeep93's local repo?

Reply:
oh ok